.github/workflows/release.yml

```diff
@@ -84,5 +102,5 @@ jobs:
         with:
           upload_url: ${{ needs.create_release.outputs.upload_url }}
           asset_path: ./rz-pm.exe
-          asset_name: rz-pm-${{ matrix.goos }}-${{ matrix.asset_arch }}.exe
+          asset_name: rz-pm-${{ matrix.goos }}-${{ matrix.goarch }}.exe
```

Comment:
revert this.

Reply:
this has been reverted back again. `asset_arch` is just the name of the arch for the file and auto updater.

Reply:
Ummm... Sorry but I didn't understand... what do you mean :?